uxarray/plot/dataarray_plot.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import matplotlib
+
+from typing import Optional, TYPE_CHECKING
+if TYPE_CHECKING:
+    from uxarray.core.dataarray import UxDataArray
+
+
+def plot(uxda, **kwargs):
+    """Default Plotting Method for UxDataArray."""
+    return raster(uxda, **kwargs)
+
+
+def raster(uxda: UxDataArray,
+           plot_height: Optional[int] = 300,
+           plot_width: Optional[int] = 600,
+           x_range: Optional[tuple] = (-180, 180),
+           y_range: Optional[tuple] = (-90, 90),
+           cmap: Optional[str] = "inferno",
+           agg: Optional[str] = "mean"):
+    """Renders a Raster Plot of an Unstructured Grid Data Variable.
+
+    Parameters
+    ----------
+    plot_width, plot_height : int, optional
+       Width and height of the output aggregate in pixels.
+    x_range, y_range : tuple, optional
+       A tuple representing the bounds inclusive space ``[min, max]`` along
+       the axis.
+    cmap: str, optional
+        Colormap used for shading
+    agg : str, optional
+        Reduction to compute. Default is "mean", but can be one of "mean" or "sum"
+    """
+    import datashader as ds
+    import datashader.transfer_functions as tf
```

Comment:
Would it be a better idea to have `import`s in the `init` so that we import them only once when `.plot` accessor is called (instead of always importing whenever a plottig function is called)?

Reply:
This approach (also in a few of our geometry functions) is how we handle optional dependencies. I can spend some time figuring out a better solution, but this is what has been working for us as of now.